ddl/db_cache_test.go

```diff
@@ -232,7 +232,14 @@ func TestCacheTableSizeLimit(t *testing.T) {
 		}
 		time.Sleep(50 * time.Millisecond)
 	}
-	require.True(t, cached)
+
+	// require.True(t, cached)
```

Comment:
Does it need to be removed?

Reply:
I think unnecessary. It's a good hint in addition to the comments below

Reply:
add some comments to be helpful to understand